stripe/resource.py

```diff
@@ -280,6 +281,36 @@ def serialize(self, previous):
 
         return params
 
+    # Override copy methods so that we can avoid having exceptions thrown as
```

Comment:
Minor comment - do you mind just cleaning up this comment (and the duplicated one below?). Either just one that indicates the problem is with copy _and_ deepcopy, or a short unique comment next to each case.